tests/PhpGenerator/Model/StructTest.php

```diff
@@ -453,4 +453,50 @@ public function resolve(): void
 
         Assert::assertExpect('Resolved.expect', $class);
     }
+
+    /**
+     * @test
+     */
+    public function generate82Class(): void
+    {
+        $class = new Struct('Sidux\PhpGenerator\Stub\RequestClass');
+        $class->setReadOnly(true);
+
+        $objectClass = new Struct('PhpGenerator\Stub\SubNamespace\PropertyTwo');
+        $constructor = $class->addConstructor();
+
```

Comment:
This test fails - to check

During the resolve(), it should find the main parent and add it in the namespace but for some reason the parent is null